specifyweb/frontend/js_src/lib/components/QueryBuilder/Components.tsx

```diff
@@ -108,6 +108,29 @@ export function SaveQueryButtons({
   );
 }
 
+export function ToggleMappingViewButton({
```

Comment:
Code is correct, but, need to reformat it with Prettier. Please configure that plugin in your IDE to have it run automatically on save